language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/references/ReferencesUtil.java

```diff
@@ -120,7 +122,20 @@ public static Optional<Symbol> getSymbolAtCursor(PositionedOperationContext cont
             return Optional.empty();
         }
 
+        Document document = srcFile.get();
         Position position = context.getCursorPosition();
+        TextRange range = TextRange.from(
+                document.textDocument().textPositionFrom(PositionUtil.getLinePosition(position)), 0);
+        NonTerminalNode nonTerminalNode = ((ModulePartNode) document.syntaxTree().rootNode()).findNode(range);
+        SyntaxKind parentKind = nonTerminalNode.parent().kind();
+
+        if (parentKind == SyntaxKind.TYPE_PARAMETER || parentKind == SyntaxKind.STREAM_TYPE_PARAMS) {
```

Comment:
Shall we include a comment mentioning the reason for searching the symbol at col-1

Reply:
Added a comment